packages/vega/test/scenegraphs/area.json

```diff
@@ -780,7 +788,9 @@
               "fill": "steelblue",
               "fillOpacity": 1,
               "interpolate": "monotone",
-              "y2": 200
+              "y2": 200,
+              "ariaRole": "graphics-symbol",
+              "ariaRoleDescription": "area"
```

Comment:
Adding aria annotations to every item makes even less sense for interpolated marks like area, line, and trail, which map to a single SVG path element. Regardless of how it is ultimately rendered out, it seems better to put information that is identical for mark items in the mark parent element of the scenegraph.

Reply:
Right. I think in this case the default is just telling a screen reader user that there is a line. It's on the chart creator to add points that are accessible.

So far, I have not focused much on making the marks accessible by default in Vega since it's not always clear what the right description is. In my Vega-Lite pull request, I am adding automatic labels for marks and I think Vega-Lite could automatically generate accessible lines (it does not right now). 

We could do this by default in Vega, though, which I think would be good but also increase the bloat. 

Maybe one option here (as you suggested above) would be to come up with some way to summarize the line in a single description. I just don't know of a good way to do that.